.github/workflows/s3Deploy.yml

```diff
@@ -0,0 +1,35 @@
+name: Deploy to S3
+
+on:
+  push:
+    branches:
+      - main
+
+# Required for authenticating with AWS IAM
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

Comment:
Use the latest version

```suggestion
        uses: actions/checkout@v3
```

It uses node 16 instead of 12
![image](https://user-images.githubusercontent.com/43217/235675883-07860252-dc46-4cfd-a1ab-9f7a45036648.png)